torch/_inductor/codegen/triton.py

```diff
@@ -151,7 +155,42 @@ class TritonOverrides(OpOverrides):
     """Map element-wise ops to Triton"""
 
     @staticmethod
-    def to_dtype(x, dtype: torch.dtype):
+    def to_dtype(x, dtype: torch.dtype, src_dtype: torch.dtype = None):
```

Comment:
Can we not get `src_dtype` at this stage? Why do we need to plumb src_dtype through the entire lowering?

Reply:
This is because only fp8 conversion has special requirement on min_elements_per_thread. Any suggestions on better ways to implement this logic?

Reply:
Yeah I was just a bit surprised that we don't have this information elsewhere during this lowering. I was thinking whether we can just get the dtype directly from `x`, but I think in general, we don't actually keep around dtype information at this stage of the lowering.